### DISCUSSION TITLE
Garmin Connect Support

### DISCUSSION THREAD

**Discussion by MWS-Koewi** [CatimaLoyalty/Android#1179] [Ideas]
Hello there!

Just as the title says, do you think would be possible to add Garmin Connect compatibility?

Thanks!

**Comment by Altonss** [comment]
I'm not sure, but probably not in the near future, unless someone wants to code it. I don't know at all what's needed/how much work it would be.
What I know is that Android Watch support is already being discussed here https://github.com/CatimaLoyalty/Android/issues/37 and WearOS here https://github.com/CatimaLoyalty/Android/issues/25, but for now  nothing planned for the near future.

**Comment by TheLastProject** [comment]
I'm not opposed to it, however, I don't have any Garmin smartwatch myself so this would need to be made by someone else.

I also don't know anything about Garmin Connect, how does it work? How would a third party app like Catima talk to it?
